tools/test-case-generators/repos.json

```diff
@@ -248,7 +248,8 @@
         "image_type_tags": [
-          "ec2-sap"
+          "ec2-sap",
+          "azure-sap-rhui"
```

Comment:
This repo has all the needed packages, the others are subsets of it as far as I can see from looking at them. I tested things by commenting out the new package that's not available and generating manifests and all the other requirements were satisfied. It should just need to have the new el8-x86_64-sap-rhui-azure snapshot added so it can pick up the new client rpm.

Reply:
Interesting 🤔 So you tried to generate manifests using all repos used by ec2-sap image and then tried to trim the number down? I can imagine that there may be some weak dependency, which may not get installed if the repo is not available. Nevertheless the list of repos used by ec2-sap image comes from historical sources and was not checked regularly, so it may be well be the case that some of them are really not needed any more.

Reply:
Actually I first tried with none of them :) According to my notes the missing packages were - `rhel-system-roles-sap, compat-sap-c++-9, compat-sap-c++-10, tuned-profiles-sap-hana, rhui-azure-rhel8-sap-ha`